README.md

```diff
@@ -1,32 +1,18 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
+# License
 
-## Getting Started
+## License
 
-First, run the development server:
+AccessibleWebDev is licensed under the MIT License - see [License](https://github.com/AccessibleForAll/AccessibleWebDev/blob/main/LICENSE)
```

Comment:
```suggestion
AccessibleForAll is licensed under the MIT License - see [License](https://github.com/AccessibleForAll/AccessibleForAll/blob/main/LICENSE)
```

Would you be able to go through and update all the links?

Reply:
Sure